Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.h

```diff
@@ -0,0 +1,22 @@
+//
+//  OAWhatsNewBottomSheetViewController.h
+//  OsmAnd
+//
+//  Created by Max Kojin on 26.11.2020.
+//  Copyright © 2020 OsmAnd. All rights reserved.
+//
+
+#import "OABottomSheetTwoButtonsViewController.h"
+
+@class OAWhatsNewBottomSheetViewController;
```

Comment:
For all new BottomSheet screens prefer using subclasses of OABaseBottomSheetViewController